lib/actions/api.js

```diff
@@ -107,15 +114,25 @@ export function routingQuery (searchId = null) {
         ({mode, params}) => ({mode, ...params})
       )
       : [{}]
-    dispatch(routingRequest({ activeItinerary, routingType, searchId, pending: iterations.length }))
-    iterations.forEach((injectedParams, i) => {
+    dispatch(routingRequest({
+      activeItinerary,
+      updateSearchInReducer,
```

Comment:
If we're alphabetizing object fields,`updateSearchInReducer` should go at the end.

Reply:
Alphabetized in a4480936984a2a1cb2d9dd6f97adc4ecd3a9a12c